gateway/ln-gateway/src/lib.rs

```diff
@@ -118,6 +118,9 @@ impl LnGateway {
             .load_federation_actors(decoders, module_gens, route_hints)
             .await;
 
+        let version = env!("GIT_HASH").to_string();
+        info!("Starting lightning gateway (version: {version}");
```

Comment:
This currently prints 

```
2023-02-08T16:44:31.786Z INFO    plugin-ln_gateway: Starting lightning gateway (version: 04a59457a253243428dab6dc4aa844d1ac4577a9
```
There's a missing closing `)` or better remove the opening `(` I guess :)

Also, since we only print the version hash, we don't need to allocate via `.to_string()`. We can just take the `&str` from the `env` macro.

Reply:
Done